lib/player.rb

```diff
@@ -0,0 +1,13 @@
+class Player
+	attr_accessor :game
+	attr_reader :token
+
+	def initialize(token)
+		@token = token
+	end
+
+	def game(game)
+		@game = game
+	end
```

Comment:
This method is a little confusing. By convention, a `game=` method would be used to set the `@game` instance variable. However, that method is already provided by `attr_accessor :game`. I would think you could remove this code.